docs/testplan/GCU-Dynamic-ACL-testplan.md

```diff
@@ -137,15 +137,53 @@ Verify that after creation, a forward ACL rule can be removed and packets matchi
 - Verify that the results of "show acl rule {rule_name}" are empty for both rule names
 
 - Verify that packets with IPs matching the removed forwarding rules are dropped on this specific port
+- 
+### Test Case # 7 - DHCP Forwarding Rule
 
-### Test Case # 7 - Scale test of ACL Table, add large amount of forward and drop rules
+#### Test Objective
+
+Verify that GCU is capable of adding a blanket DHCP forwarding rule, and that traffic follows these rules accordingly
+
+#### Testing Steps
+
+- Use GCU to create a blanket DHCP packet forwarding rule
+
+- Use GCU to create a blanket drop rule on a port
+
+- Verify that all operations were successful
+
+- Verify that DHCP packets are forwarded
+
```

Comment:
Please be noted that the dhcp request will be relayed by the dhcp_relay service on T0, so the dst_ip of the DHCP packet being forwarded by T0 will be modified to one of the dhcp_helper address.